packages/agoric-cli/lib/start.js

```diff
@@ -28,6 +76,51 @@ export default async function startMain(progname, rawArgs, powers, opts) {
     return pr;
   };
 
+  let keysSpawn;
```

Comment:
I know its a matter of taste, but I have to say I could read this better if it were simply:

```js
function keysSpawn(args, ...rest) {
  if (opts.sdk) {
    return nondockerstuff...;
  } else {
    return dockerstuff..;
  }
}
```